twill-yarn/src/main/java/org/apache/twill/yarn/ResourceReportClient.java

```diff
@@ -52,12 +54,16 @@
   public ResourceReport get() {
     for (URL url : resourceUrls) {
       try {
-        Reader reader = new BufferedReader(new InputStreamReader(url.openStream(), Charsets.UTF_8));
-        try {
+        HttpURLConnection urlConn = (HttpURLConnection) url.openConnection();
```

Comment:
Is this change related to Netty upgrade?

Reply:
Not entirely. This is to enable compression when fetching the report.